lib/api/_loaders/_command-loader.js

```diff
@@ -33,8 +34,9 @@ class BaseCommandLoader extends BaseLoader {
   getCommandOptions() {
     const redact = this.module.RedactParams || false;
     const alwaysAsync = this.module.alwaysAsync || false;
+    const isTraceable = (Utils.isFunction(this.module.isTraceable)? this.module.isTraceable() : this.module.isTraceable) || false ;
```

Comment:
whhy do we need to check if it's a function? I think everywhere it's a get property?

Reply:
For custom commands